runtime/monitoring/syslog.go

```diff
@@ -1,6 +1,6 @@
 // +build !windows
 
-package daemon
+package monitoring
```

Comment:
you forgot this in syslog_null...  I've done this in https://github.com/taskcluster/taskcluster-worker/pull/259